Dockerfile

```diff
@@ -9,11 +9,7 @@ RUN apt-get update && \
     apt-get install -y libffi-dev --force-yes && \
     pip install gunicorn &&\
 	pip install --upgrade "git+http://github.com/rackerlabs/graphite-api.git@george/fetch_multi_with_patches" &&\
```

Comment:
Do we still need to rely on George's branch or are the changes merged with the master?